app/models/judge.rb

```diff
@@ -3,6 +3,11 @@ class Judge < ApplicationRecord
   belongs_to :region
   has_many :activities
 
+  default_scope { active }
```

Comment:
In this case, setting the default_scope prevents admins from seeing a hidden judge that has been assigned to an activity.  For example, when a community admin is looking at a Friend record, there is a section for past Activities, and the default scope results in a nil result for a hidden judge on one of those activities.  Here's a quick verification I did in rails console for reference:
<img width="909" alt="Screen Shot 2019-11-17 at 9 01 14 PM" src="https://user-images.githubusercontent.com/3056607/69019698-b9beb080-097f-11ea-8adb-c15a899a2d58.png">
I think it would be best to remove this as the default_scope and rely on the 'active' scope explicitly for the judge dropdowns.